index.html

```diff
@@ -35,5 +35,113 @@ <h2 class="title">I'm a Software Developer</h2>
+    </div>
+    <div class="background-card-1">
+      <div class="project-title">Profesional Art Printing Data</div>
+      <div class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+      <div class="buttons">
+        <div class="simple-button">html</div>
+        <div class="simple-button">bootstrap</div>
+        <div class="simple-button">Ruby</div>
+      </div>
+      <div class="project-button">See Project</div>
+    </div>
+    <div class="background-card-1">
+      <div class="project-title">Profesional Art Printing Data</div>
+      <div class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+      <div class="buttons">
+        <div class="simple-button">html</div>
+        <div class="simple-button">bootstrap</div>
+        <div class="simple-button">Ruby</div>
+      </div>
+      <div class="project-button">See Project</div>
+    </div>
+    <div class="background-card-1">
+      <div class="project-title">Profesional Art Printing Data</div>
+      <div class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+      <div class="buttons">
+        <div class="simple-button">html</div>
+        <div class="simple-button">bootstrap</div>
+        <div class="simple-button">Ruby</div>
+      </div>
+      <div class="project-button">See Project</div>
+    </div>
+    <div class="background-card-1">
+      <div class="project-title">Profesional Art Printing Data</div>
+      <div class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+      <div class="buttons">
+        <div class="simple-button">html</div>
+        <div class="simple-button">bootstrap</div>
+        <div class="simple-button">Ruby</div>
+      </div>
+      <div class="project-button">See Project</div>
+    </div>
+    <div class="background-card-1">
+      <div class="project-title">Profesional Art Printing Data</div>
+      <div class="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+      <div class="buttons">
+        <div class="simple-button">html</div>
+        <div class="simple-button">bootstrap</div>
+        <div class="simple-button">Ruby</div>
+      </div>
+      <div class="project-button">See Project</div>
+    </div>
+  </section>
+  <section class="about-me">
+    <div class="title">About Me</div>
+    <div class="description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
+    <div class="button">Get My Resume</div>
```

Comment:
- [ ] Kindly use button or `<a>` for buttons and follow the button styling: Active and Hover.
![image](https://user-images.githubusercontent.com/53402600/176794552-51077e9f-f96a-4690-8af4-145e04c725f8.png)